js/tests/unit/dropdown.spec.js

```diff
@@ -54,6 +54,36 @@ describe('Dropdown', () => {
       expect(dropdown.toggle).toHaveBeenCalled()
     })
 
+    it('should allow click events to bubble', () => {
+      fixtureEl.innerHTML = [
+        '<div class="dropdown">',
+        '  <button class="btn dropdown-toggle" data-bs-toggle="dropdown">Dropdown</button>',
+        '  <div class="dropdown-menu">',
+        '    <a class="dropdown-item" href="#">Secondary link</a>',
+        '  </div>',
+        '</div>'
+      ].join('')
+
+      const btnDropdown = fixtureEl.querySelector('[data-bs-toggle="dropdown"]')
+      const clickListener = jasmine.createSpy('clickListener')
+      const delegatedClickListener = jasmine.createSpy('delegatedClickListener')
+
+      btnDropdown.addEventListener('click', () => clickListener())
+
+      document.addEventListener('click', event => {
+        if (!event.target.matches('[data-bs-toggle="dropdown"]')) {
+          return
+        }
```

Comment:
You can ignore this check here since you are clicking the toggle button only. :thinking: 
Expecting the callback method to have been called would be enough here IMO
```js
btnDropdown.addEventListener('click', clickListener)
document.addEventListener('click', delegatedClickListener)
```